pom.xml

```diff
@@ -617,7 +617,7 @@
         <version.org.testcontainers>1.16.0</version.org.testcontainers>
         <version.org.testng>7.4.0</version.org.testng>
         <version.org.wildfly.arquillian>5.0.0.Alpha3</version.org.wildfly.arquillian>
-        <version.org.wildfly.core>19.0.0.Beta14</version.org.wildfly.core>
+        <version.org.wildfly.core>19.0.0.Beta15</version.org.wildfly.core>
```

Comment:
@jasondlee you can remove this from this commit since it doesn't belong to it. Besides that, changes LGTM, @fl4via Could you approve on your side?

Reply:
Done.